utils/payload_utils.py

```diff
@@ -0,0 +1,32 @@
+import re
+
+
+class PayloadUtils:
```

Comment:
I have created this new class to extract data from the payload, the description field in test rail and create new colummns with the data we will be interested in.